### PR TITLE
chore(www) show reading time ceil

### PR DESCRIPTION
**Pull request by hrstmr**
Closes #

## 🎯 Changes

Added Math.ceil() to pass the reading time to og Image generator as an int instead of a decimal, making is consistent with the visible reading time on the blog page

## ✅ Checklist

- [x] I have followed the steps listed in the [Contributing guide](https://github.com/trpc/trpc/blob/main/CONTRIBUTING.md).
- [ ] If necessary, I have added documentation related to the changes made.
- [ ] I have added or updated the tests related to the changes made.

### CODE REVIEW

**Comment by hrstmr** [comment]
Here is a screenshot of the og image showing `3.29 min read`
while the blog shows the read time as `4 min read`

image

image

**Review by KATT — CHANGES_REQUESTED** [review]
Do it within the component instead please :)

**Comment by hrstmr** [comment]
> Do it within the component instead please :)

Moved the Math.ceil logic to OG component as requested 😄

File: www/og-image/pages/api/blog.tsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ export default async (req: Request) => {
             </div>
             <p tw="text-3xl leading-snug text-zinc-300">{props.description}</p>
             <p tw="text-2xl text-blue-500 font-semibold leading-3">
-              {props.readingTimeInMinutes} min read
+              {Math.ceil(props.readingTimeInMinutes)} min read
```
Comment (KATT) [inline www/og-image/pages/api/blog.tsx:50]:
```suggestion
              {Math.round(props.readingTimeInMinutes)} min read
```

**Review by KATT — APPROVED** [review]

**Pull request merged**